jena-fuseki2/jena-fuseki-ui/src/services/fuseki.service.js

```diff
@@ -142,11 +142,11 @@ class FusekiService {
       })
     } catch (error) {
       if (error.response) {
-        if (error.response.status !== 200) {
-          if (error.response.status === 409) {
+        if (error.response.statusCode !== 200) {
+          if (error.response.statusCode === 409) {
             throw new Error(`failed to create dataset "${datasetName}", reason: there is another dataset with the same name`)
           }
-          throw new Error(`failed to create dataset "${datasetName}" with type ${datasetType}, reason: HTTP status: "${error.response.status}", message: ${error.response.statusText}`)
+          throw new Error(`failed to create dataset "${datasetName}" with type ${datasetType}, reason: HTTP status: "${error.response.statusCode}", message: ${error.response.statusText}`)
```

Comment:
Probably a bug :disappointed_relieved: `status` is a function, so it would never be `200`, `409`. The IDE warned me about the type mismatch, and looking at the code I realized we should have used `statusCode`. Fixed, and tests updated too.

The code would still fail to create the dataset, but I believe it was just not displaying a useful message.